webapp/src/stories/Button.stories.ts

```diff
@@ -0,0 +1,52 @@
+import type { Meta, StoryObj } from '@storybook/vue3';
+
+import Button from './Button.vue';
+
+// More on how to set up stories at: https://storybook.js.org/docs/vue/writing-stories/introduction
+const meta = {
+  title: 'Example/Button',
+  component: Button,
+  // This component will have an automatically generated docsPage entry: https://storybook.js.org/docs/vue/writing-docs/autodocs
+  tags: ['autodocs'],
```

Comment:
Let's remove storybook for now, we will essentially be building entire pages using vuetify components, so I don't really see a use case right now, as we won't create a lot of small components.

Reply:
Agreed